src/appModel.ts

```diff
@@ -85,8 +91,6 @@ export class AppModel {
                 }
             );
         }
-
-        this.revertUIToWatchingStatusNow()
```

Comment:
Re-add this. [ref](https://github.com/glenn2223/vscode-live-sass-compiler/pull/3#issuecomment-748591053)